### ISSUE TITLE
Improve assessment for the dissimilar glyph pairs

### ISSUE THREAD

**Issue by MariannaPaszkowska** [behdad/halfkern#2]
Hey there! Halfkern is currently providing some overkern suggestions for the "-J" and "C-". Maybe there is a way to improve this.

**Comment by behdad** [comment]
Yeah I realized that with a Serif font. I think I like the "C-" one...

Note that of you pass `-l` to the `kern_language.py` it will only process letters, no punctuation.

**Comment by behdad** [comment]
> overkern

I need to change algorithm slightly to handle those better.
